.template-helpers/variant.md

```diff
@@ -0,0 +1,7 @@
+# Image Variants
+
+The `%%REPO%%` images come in many flavors, each designed for a specific use case.
```

Comment:
IMO that's kind of deep in the semantics of the language, so I'd prefer we err on the side of pointing out and reinforcing to users that this really isn't just "one `ruby` image", but is in fact a collection of multiple related images.

As a concrete example, `ruby:slim` and `ruby:onbuild` are _very_ different images, and they don't even share a common ancestor besides `debian` itself.